src/Microsoft.AspNetCore.Mvc.Core/ModelBinding/Binders/BodyModelBinder.cs

```diff
@@ -177,9 +177,27 @@ public BodyModelBinder(IList<IInputFormatter> formatters, IHttpRequestStreamRead
                     bindingContext.ModelState.AddModelError(modelBindingKey, message);
                 }
             }
-            catch (Exception ex)
+            catch (Exception exception)
```

Comment:
Add a catch block ahead of this one for `InputFormatterException`. We always want to handle those.